src/pages/TodoItemsCompletePage.tsx

```diff
@@ -1,33 +1,33 @@
-import { useTodoCards } from "../hooks";
 import { TodoItemCard } from "../components/todo-item-card";
+import { useTodoList } from "../hooks";
 
 const TodoItemsCompletePage = () => {
-  const { todoCards } = useTodoCards();
-
-  return (
-    <>
-      <section className="container mx-auto">
-        <h1 className="text-3xl font-bold mb-4">Task Completed</h1>
-
-        {
-          <div className="grid grid-cols-4 gap-4">
-            {todoCards
-              .filter((c) => c.isCompleted == true)
-              .map((card) => (
-                <TodoItemCard
-                  isImportant={card.isImportant}
-                  key={card.id}
-                  todoItemCard={card}
-                  onCardComplete={() => undefined}
-                  onCardDelete={() => undefined}
-                  onCardEdit={() => undefined}
-                />
-              ))}
-          </div>
-        }
-      </section>
-    </>
-  );
+  // const {
+  //   query: { list },
+  // } = useTodoList();
+  // return (
+  //   <>
+  //     <section className="container mx-auto">
+  //       <h1 className="text-3xl font-bold mb-4">Task Completed</h1>
+  //       {
+  //         <div className="grid grid-cols-4 gap-4">
+  //           {list
+  //             ?.filter((c) => c.isCompleted == true)
+  //             .map((card) => (
+  //               <TodoItemCard
+  //                 isImportant={card.isImportant}
+  //                 key={card.id}
+  //                 todoItemCard={card}
+  //                 onCardComplete={() => undefined}
+  //                 onCardDelete={() => undefined}
+  //                 onCardEdit={() => undefined}
+  //               />
+  //             ))}
+  //         </div>
+  //       }
+  //     </section>
+  //   </>
+  // );
 };
```

Comment:
Why it was commented?

Reply:
I have no clue why todo list does not show on pages. The reason was in re-calling **setQuery** in the try catch construction. Fixed. But forget to uncomment.